sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/HiveThriftServer2Suites.scala

```diff
@@ -1122,62 +1123,6 @@ object ServerMode extends Enumeration {
   val binary, http = Value
 }
 
-abstract class HiveThriftJdbcTest extends HiveThriftServer2Test {
```

Comment:
Do we need to change this?

Reply:
Reflecting this is to bring its functions to the superclass, which is used to add a probe to check the server is responsive after started. And If It is not it can retry to reduce flakiness and dump the server log to `unit-test.log` for us to see what is wrong actually

Reply:
FYI, fot flaky test like https://amplab.cs.berkeley.edu/jenkins/job/SparkPullRequestBuilder/132602/testReport/org.apache.spark.sql.hive.thriftserver/ThriftServerWithSparkContextInHttpSuite/_It_is_not_a_test_it_is_a_sbt_testing_SuiteSelector_/

Jenkins can have more chances to retry and provide more logs for us to debug

Reply:
Got it.